crypto/account.go

```diff
@@ -0,0 +1,48 @@
+package crypto
+
+/*
+Convenience object for dealing with keys/addresses
+*/
+
+import (
+	"crypto/ecdsa"
+	"encoding/hex"
+	"strings"
+)
+
+func CreateAccount() (*Account, error) {
+	key, err := GenerateKey()
+	if err != nil {
+		return nil, err
+	}
+	return &Account{
+		key: key,
+	}, nil
+}
+
+func ParsePrivateKey(pkHex string) (*Account, error) {
+	fromPK := strings.TrimPrefix(pkHex, "0x")
+	key, err := HexToECDSA(fromPK)
+	if err != nil {
+		return nil, err
+	}
+	return &Account{
+		key: key,
+	}, nil
+}
+
+type Account struct {
+	key *ecdsa.PrivateKey
+}
```

Comment:
Is `package crypto` the best place for this? What about `accounts`?

Reply:
I'm not sure. It's really just a convenience class for the crypto functions. Think it would be better to expand  `accounts.Account`?

Reply:
Eh maybe just use another term instead of 'Account', like 'Key', and leave it here?

Reply:
Done